rotkehlchen/tests/unit/decoders/test_stakedao.py

```diff
@@ -426,3 +426,63 @@ def test_withdraw_bsc(binance_sc_inquirer, binance_sc_accounts, stakedao_gauges)
         counterparty=CPT_STAKEDAO,
         address=string_to_evm_address('0xE36c375AD28C0822ab476Bd62A8BEAd88d9a4e34'),
     )]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x811e8f6d80F38A2f0f8b606cB743A950638f0aD4']])
+def test_claim_rewards(arbitrum_one_inquirer, arbitrum_one_accounts):
+    tx_hex = deserialize_evm_tx_hash('0x7ce0becce93ff093c51f33c2012fd3d2dfa5118cedb1c60d9cad339ceb3e4ae4')  # noqa: E501
```

Comment:
```suggestion
    tx_hash = deserialize_evm_tx_hash('0x7ce0becce93ff093c51f33c2012fd3d2dfa5118cedb1c60d9cad339ceb3e4ae4')  # noqa: E501
```
nitpick, but this should probably be `tx_hash`